### ISSUE TITLE
Queue for transfer in listview bulk actions

### ISSUE THREAD

**Issue by bjarnef** [umbraco/Umbraco.Deploy.Issues#140]
With the new option to transfer members it would be nice to allow queue for transfer in bulk actions in listview.
I noticed the same is valid to content and media listview.

For now we either have to transfer parent with child nodes or add each entity to queue individually.

Umbraco v10.1.1
Deploy v10.0.2

**Comment by hfloyd** [comment]
Yes! I want this also! Would it be possible to add a "Queue for Transfer" button to the selection bar?

![image](https://user-images.githubusercontent.com/1252593/216410416-a88c4bc8-2dd2-4101-abb5-dccc482a5b53.png)

**Comment by AndyButland** [comment]
I recall having a look at this... but it's tricky unfortunately, and I don't see a way to do it.  The issue is that [these buttons are basically hard-coded](https://github.com/umbraco/Umbraco-CMS/blob/contrib/src/Umbraco.Web.UI.Client/src/views/propertyeditors/listview/listview.html#L157), with just some logic to show or hide them based on permissions.  There doesn't seem to be an extension point for adding new ones.

**Comment by hfloyd** [comment]
So, I guess changes to Umbraco Core would be required for this to be possible, then?

**Comment by AndyButland** [comment]
I think so.  Hopefully the new back-office will provide an extension point to be able to include such things.
